index.html

```diff
@@ -0,0 +1,331 @@
+</dl>
+
+The version tag **must** satisfy the [BNF grammar for `‹version tag›`](#backus-naur-form-for-versions-and-version-tags).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>prefix</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@using` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@using` **should** prefix any schema elements they introduce. The prefix **must** be a string of characters valid within GraphQL typenames, with the exception of `"_"`.
+
+Specs **should** define any new schema elements with names that start with `prefix_`.
+
+```graphql
+#::[Using the `@using` directive with a spec's default prefix]
+schema @using(spec: "https://spec.example.com/example/v1") {
+  query: Query
+}
+
+type User {
+  name: String @example_someDirective
+               # prefixed directive from the example spec
+}
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+### Elements which should be prefixed
+Specs **should** attach their prefix to:
+- the names of any object types, interfaces, unions, enums, or input object types defined in the spec
+- the names of any fields the spec introduces on *foreign types* defined in a different spec
+- the names of any arguments the spec introduces on *foreign types* defined in a different spec
+- the names of any directives introduced in the spec
```

Comment:
I noticed there's an omission of enum values on foreign enums. I'm a bit curious about the reasoning here (I imagine some of it might be the `SNAKE_CASE` problem, although that could be alleviated by using double-underscore after the prefix as suggested above).

Reply:
@glasser and I had a bit of a discussion about this. enums are tricky. Adding extra nullable fields and arguments is straightforward—you can just ignore any fields / arguments you don't understand. Most code will do this automatically—it won't look for unknown values and thus, won't find them. That's really not the case with enums. It is, in fact, a lot harder to come up with a good case for extending an enum after it's been defined, because things that expect the pre-extension enum won't know what to do with extended values.

It's also difficult to imagine what you could accomplish with extension values that you can't accomplish by putting a directive on arbitrary document-defined enum values.

At the same time, I don't know if we want to say _NEVER DO THIS_? But I'm fine with leaving it in this undefined territory for 0.1 and revisiting it come 0.2